### PR TITLE
add ulas chain

### PR DESCRIPTION
**Pull request by armatrix**
ULAS network(AKA https://muses.network, rename due to the promotion) is a multi-chain social protocol built on an open social graph and rating system. The ecosystem application on ULAS will bring data sovereignty, higher financial returns, and better experience to Web3.0 citizens.

### CODE REVIEW

File: ulas/chain.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      {
+        "address": "http://65.49.204.199:9090",
+        "provider": "UlasNetwork"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "mintscan",
+      "url": "https://mintscan.io/ulasnetwork",
```
Comment (JeremyParish69) [inline ulas/chain.json:42]: This doesn't work on Mintscan. Is this supposed to be added soon? Or is there no explorer?

File: ulas/chain.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "ULAS",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "ULAS",
+  "chain_id": "ulas",
+  "bech32_prefix": "ulas",
```
Comment (JeremyParish69) [inline ulas/chain.json:8]: Please register this prefix to SLIP 173
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: ulas/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+          "denom": "ulas",
+          "exponent": 6
+        }
+      ],
+      "base": "uulas",
+      "name": "ULAS Network",
+      "display": "ULAS",
+      "symbol": "ULAS",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/ulasnetwork/chain-config/main/ulas/images/logo.png"
```
Comment (JeremyParish69) [inline ulas/assetlist.json:22]: Please re-point this URI to the local repo

File: ulas/chain.json (hunk 1 above)
Comment (armatrix) [reply ulas/chain.json]: updated to https://testnet-explorer.ulas.network

File: ulas/chain.json (hunk 2 above)
Comment (armatrix) [reply ulas/chain.json:8]: Done

File: ulas/assetlist.json (hunk 3 above)
Comment (armatrix) [reply ulas/assetlist.json]: Done

**Comment by armatrix** [comment]
> Hi @armatrix , I just noticed this is a testnet. The Chain registry has a /testnets/ folder for testnets--could you please move it to there?

Done. Tks.

**Review by JeremyParish69 — APPROVED** [review]

**Pull request merged**